piccolo_api/fastapi/endpoints.py

```diff
@@ -506,6 +506,37 @@ def modify_signature(
                     ]
                 )
 
+            parameters.extend(
+                [
+                    Parameter(
+                        name="__range_header",
+                        kind=Parameter.POSITIONAL_OR_KEYWORD,
+                        annotation=bool,
+                        default=Query(
+                            default=False,
+                            description=(
+                                "enable to add range-header respons header"
+                            ),
+                        ),
+                    )
+                ]
+            )
+            parameters.extend(
+                [
+                    Parameter(
+                        name="__range_header_name",
+                        kind=Parameter.POSITIONAL_OR_KEYWORD,
+                        annotation=str,
+                        default=Query(
+                            default=None,
+                            description=(
+                                "object name in range-header reponse header"
```

Comment:
For the description, how about:

`Specify the object name in the Content-Range response header (defaults to the table name).`